### DISCUSSION TITLE
Is Bourgeois Global inverter running with OpenDTU ?

### DISCUSSION THREAD

**Discussion by Forhorse88** [tbnobody/OpenDTU#647] [Q&A] [answered]
Hello, i have un Bourgeois Global by Hoymiles MOBG-800HM sold as a Hoymiles HM-800 inverter but it don't seem to communicate with Open DTU.
The serial of the inverter is 114184232723 and openDTU find a "HM-600, HM-700, HM-800" inverter.

The console of OpenDTU say 

> 10:04:31.221 > TX PowerControl Channel: 75 --> 51 84 23 27 23 80 19 91 60 81 00 00 B0 01 AA 
> 10:04:33.258 > RX Period End
> 10:04:33.258 > All missing
> 10:04:33.258 > Nothing received, resend whole request

on all channels again and again but nothing else happens
On the inverter, the led is flashing green. The inverter and the nRF24 antenna are at 6 apart, barrier-free (outdoor direct wiew)

Did Bourgeois Global inverters not running  with OpenDTU ?
Did I miss something else?

Thank you

**Comment by Forhorse88** [comment, marked as answer]
I'm replying to myself...
Yes Bourgeois Global 800HM inverter are compatible with openDTU
I had buil a new hardware with a Wemos ESP32 and the same nRF24 module and it's running "out of the box" 
I must have a hardware issue with WT32-ETH01 module (maybe power supply ? mybe pinout ?)

**Reply by Forhorse88** [reply]
Now it's running with the same WT32-ETH01 module, the same nRF24 module and the same pinout but with an improved scheme for the power supply.
Did I make a connection error the first time?
was the power supply bad ?
I don't know, but now it's running weel !

**Reply by beneva52** [reply]
Yep, a good power supply is the key.
The nRF24 module needs juice and alters ESP32 performances.

**Reply by stefan1967** [reply]
Hi Forhorse88,

I would also like to use an WT32, did you use a predefined Build file or did you creat your own build?
Could you provide me the pin layout you used, my Wt 32 looks different then the ESP32, so I am not sure at the moment how to connect the radio board. thank you much!

**Reply by Forhorse88** [reply]
I'm using predefined build. The pinout is not critical and you can defin your own as needed.

`"nrf24": {
            "miso": 4,
            "mosi": 2,
            "clk": 32,
            "irq": 33,
            "en": 14,
            "cs": 15
}`

and 
`"nrf24": {
            "miso": 2,
            "mosi": 15,
            "clk": 33,
            "irq": 4,
            "en": 32,
            "cs": 14
        }`

Both work fine (i'm using the second for a simplified PCB layout)
